src/carbonado.rs

```diff
@@ -78,6 +83,43 @@ mod server {
         Ok((filepath, body))
     }
 
+    pub async fn auctions_store(
```

Comment:
I believe so, but I feel like it will lose some of its purpose. I centralized in this file all the calls that are sometimes made by the client and sometimes made by the server.